eZ/Bundle/EzPublishCoreBundle/Resources/views/default/embed.html.twig

```diff
@@ -0,0 +1,3 @@
+{% for field in content.FieldsByLanguage %}
+    {{ ez_render_field(content, field.fieldDefIdentifier) }}
+{% endfor %}
```

Comment:
> legacy template checked if node.url_alias exists before trying to show the link. will that check be necessary here too?

I guess only relevant check is to see if content info has main location id, cause if not it will have to generate system url.